packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/sdk/test/arm-test/review/arm-networkanalytics.api.md

```diff
@@ -442,6 +449,7 @@ export class NetworkAnalyticsApi {
 // @public
 export interface NetworkAnalyticsApiOptionalParams extends ClientOptions {
     apiVersion?: string;
+    cloudSetting?: AzureClouds;
```

Comment:
would it make sense to support passing in the enum values directly via string literals?

Example of what I mean: https://www.typescriptlang.org/play/?#code/KYDwDg9gTgLgBMAdgVwLZwIIC9lWAYQBsJkATAZzgG8BYAKDkcwC0BVAJQFEB9fACQCSAOQy8AMgHlWAETgBeOACIMbLr0EjxU6YoA09Jiw48ACqwBCYgfi0z5Slce5nL12zv0MmjtawDK3ADiEgBqnOxCALKcQgAq9sqqPP5BoeFRMbGK9AC+9PQwAJ5gwJg4eH7IYJCwwKREJBT2AAYAJFTYuATEZOQ5zQDc+XQAZsiIAMYwAJYQiHAjEBAAFABuAIaEyMAAXGVdldXQMHUNvQCU1MN5dPSLK514ZxQAdD48-MKi+JIy50OjJbLRJOT6aH7aRT-IA

Since this widens `cloudSetting` we don't have to decide this now, it could be a future enhancement if desired.